src/common/wallet-state.ts

```diff
@@ -728,6 +734,11 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
     }
   }
 
+  useRecurringTimeout(async () => {
+    await refreshSyncStatus()
+    rendererLog.debug(`sync status: ${syncStatusOption}`)
+  }, 3000)
+
   return {
```

Comment:
One comment actually, I would not export `refreshSyncStatus` from wallet state now. It should be responsibility of the wallet state to update this sync status, right?

Reply:
You're right! Fixing